hexrd/rotations.py

```diff
@@ -92,16 +92,8 @@ def arccosSafe(temp):
         print("attempt to take arccos of %s" % temp, file=sys.stderr)
         raise RuntimeError("unrecoverable error")
 
-    gte1 = temp >= 1.0
-    lte1 = temp <= -1.0
-
-    temp[gte1] = 1
-    temp[lte1] = -1
-
-    ang = np.arccos(temp)
-
-    return ang
-
+    temp = np.clip(temp, -1.0, 1.0)
+    return np.arccos(temp)
```

Comment:
Can probably combine these lines

Reply:
I also really don't like that the argument name is `temp`. Should be something more useful, like `cos_angle` maybe?

Reply:
Fixed